openwisp_monitoring/device/static/monitoring/js/chart.css

```
@@ -0,0 +1,156 @@
+  display: none;
+  border: 0 none;
+}
+#ow-chart-contents > div {
+  position: relative;
+}
+.chart-help {
+  display: inline-block;
+  width: 22px;
+  height: 21px;
+  text-indent: -9999px;
+  background: url(../../admin/img/icon-unknown.svg) 0 0 no-repeat;
```

Stylelint (17.4.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)